src/main/java/it/gov/pagopa/cgn/portal/model/DiscountEntity.java

```diff
@@ -82,6 +82,9 @@
     @Column(name = "landing_page_referrer", length = 100)
     private String landingPageReferrer;
 
+    @Column(name = "visible_on_eyca")
```

Comment:
we can add @NotNull annotation here

Reply:
fixed in https://github.com/pagopa/cgn-onboarding-portal-backend/pull/84/commits/eea04e476a3b7f0d35f7dc15faabe462f9483ec9